webcommon/javascript2.vue/test/unit/data/testfiles/embedding/vue_embedding_01.vue

```diff
@@ -0,0 +1,51 @@
+<!--
```

Comment:
For tests please include the ASF header. There is no need to skip it and we should carry it where possible. This comment applies for other test files too.